src/GUI/Shared/Header.razor.css

```diff
@@ -0,0 +1,51 @@
+﻿.header {
+    width: 100%;
+    height: 75px;
+
+    background-color: #7AC7FF;
+    display: flex;
+    align-items: center;
+}
+
+.header input {
+    width: 23%;
+    height: 50%;
+
+    background: #FFFFFF;
+    border: 1px solid #FFFFFF;
+    box-sizing: border-box;
+    border-radius: 5px;
+
+    margin-left: 9%;
+}
+
+.header img {
```

Comment:
you can use the next class for img <img src="..." alt="..." class="rounded">
check here https://getbootstrap.com/docs/4.0/utilities/borders/

Reply:
Fixed